active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+Breadcrumb control
+===
+
+# Background
+
+A "Breadcrumb" UX pattern provides an way to show a current position in context.
+For example in Windows File Explorer the current directory is shown with a breadcrumb,
+where clicking on any node takes you to that ancestor directory:
+
+![File Explorer breadcrumb example](images/file-explorer-breadcrumb.jpg)
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+This is not the 'history' of the user's navigation, but the direct path of pages or folders from
+the root node to their current position. 
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 the nodes have no option to open a drop-down menu, typically used to show the node's siblings.
+V2 will add this. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
```

Comment:
To me, each item is a "Breadcrumb" and the overall control is a "breadcrumb bar" / "breadcrumb path" / "breadcrumb trail" / "breadcrumb list".

DevExpress calls it a [BreadcrumbControl](https://docs.devexpress.com/WPF/401002/controls-and-libraries/navigation-controls/breadcrumb-control).

Telerik calls it a [BreadCrumb](https://www.telerik.com/products/winforms/breadcrumb.aspx) (capital C).

Actipro calls it a [Breadcrumb](https://www.actiprosoftware.com/products/controls/wpf/navigation).

Fluent UI calls it a [Breadcrumb](https://fluentsite.z22.web.core.windows.net/0.52.0/components/breadcrumb/definition).

So it seems I am the weirdo.

Reply:
I looked at the casing of the "C" too, and breadcrumb is indeed a word.

We also have a naming guideline against one-word names, and so we could call this BreadcrumbControl or BreadcrumbBar (aligns with ToolBar). But the proposal here is that this is a standard enough term, and won't conflict with other APIs.